extensions/vscode/webviews/homeView/src/components/tree/TreeItem.vue

```diff
@@ -151,6 +155,7 @@ const toggleExpanded = () => {
 
     .tree-item-label-container {
       flex: 1;
+      display: flex;
```

Comment:
Had to bring in flex for the label container so that the input in the description `span` could be displayed the way we want. This required bringing down the

```css
min-width: 0;
overflow: hidden;
text-overflow: ellipsis;
```

CSS for dealing with overflowing with ellipsis.